src/artemis/parameters/schemas/experiment_schemas/grid_scan_params_schema.json

```diff
@@ -46,7 +46,10 @@
         },
         "omega_start": {
             "type": "number"
-        }
+        },
+        "x_axis": {},
+        "y_axis": {},
+        "z_axis": {}
```

Comment:
Should: These are derived values from the other parameters and so I don't think they should be in here

Reply:
Yh but what happens if you do specify them as something that doesn't match the above params? It's confusing as an API

Reply:
ah yeah alright, that's a good point. turns out it's not actually hard to exclude them, I got the wrong idea from some older threads on the topic